docs/getting-started/setup/macos.md

```diff
@@ -210,6 +262,4 @@ To install [podman-compose](https://github.com/containers/podman-compose), run:
 
 ```shell
 brew install podman-compose
```

Comment:
I think we can remove this now? I'm pretty sure more recent podman versions contain `compose` as a command similar to newer docker versions.